_data-prepper/pipelines/configuration/processors/otel-trace-group.md

```diff
@@ -0,0 +1,84 @@
+---
+layout: default
+title: OTel trace group
+parent: Processors
+grand_parent: Pipelines
+nav_order: 45
+---
+
+# OTel trace group
+
+The `OTel trace group` processor completes missing trace group related fields in the collection of [span](https://github.com/opensearch-project/data-prepper/blob/834f28fdf1df6d42a6666e91e6407474b88e7ec6/data-prepper-api/src/main/java/org/opensearch/dataprepper/model/trace/Span.java) records by looking up the OpenSearch backend. The `OTel trace group` processor identifies the missing trace group information for a `spanId` by looking up the relevant fields in its root `span` stored in OpenSearch.
+
+## OpenSearch
+
+See the following example `YAML` configuration file for the `OTel trace group` processor when you connect to an OpenSearch cluster using your username and password:
+
+``` YAML
+pipeline:
+  ...
+  processor:
+    - otel_trace_group:
+        hosts: ["https://localhost:9200"]
+        cert: path/to/cert
+        username: YOUR_USERNAME_HERE
+        password: YOUR_PASSWORD_HERE
+```
+
+See [OpenSearch security](https://github.com/opensearch-project/data-prepper/blob/834f28fdf1df6d42a6666e91e6407474b88e7ec6/data-prepper-plugins/opensearch/opensearch_security.md#L4) for a more detailed explanation of which OpenSearch credentials and permissions are required and how to configure those credentials for the `OTel trace group` processor.
+
+### Amazon OpenSearch Service
+
+See the following example `YAML` configuration file for the `OTel trace group` processor when you use the Amazon OpenSearch Service:
+
+``` YAML
+pipeline:
+  ...
+  processor:
+    - otel_trace_group:
+        hosts: ["https://your-amazon-opensearch-service-endpoint"]
+        aws_sigv4: true
+        cert: path/to/cert
+        insecure: false
+```
+
+## Configuration
+
+You can configure the Amazon OpenSearch Service with the following options.
```

Comment:
@dlvenable @asifsmohammed Are we talking about the OTel processor or OpenSearch? This is a little confusing and could benefit from some more clarification.

Reply:
We are talking about OTel Trace Group processor here. It's confusing because it uses the same configuration as OpenSearch sink.